nservicebus/sqlserver/design.md

```diff
@@ -3,6 +3,14 @@ title: SQL Server Transport
 summary: The design of SQL Server transport
 tags:
 - SQL Server
+related:
+- /nservicebus/sqlserver/usage

```

Comment:
related cannot start with a '/'

If you update to the latest markdownpad hack it will validate this for you